### PR TITLE
G3T-22: Command-line arguments and pull down network diagram

### PR DESCRIPTION
**Pull request by ethanizen**
This PR will add the following:
* Parse command-line arguments
* Use trivium-cli to pull down diagram inputted by user

### CODE REVIEW

**Comment by zacanderson** [comment]
+1, code retains original functionality with a much nicer wrapper that will allow for a Distill package to be installed on the user's machine. Needs to be extended for networkX when this is implemented.

**Review by zacanderson — APPROVED** [review]
+1, code retains original functionality with a much nicer wrapper that will allow for a Distill package to be installed on the user's machine. Needs to be extended for networkX when this is implemented.

File: distill/distill.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    parser = argparse.ArgumentParser(description='''Distill tool for Trivium\n
+    Required Input: Distill -m -d [model_name] [diagram_name] --> pulls down network model\n
+    Optional Input: Distill -m -d -n [model_name] [diagram_name] [nessus_list] --> performs and outputs Nessus Scan''', formatter_class=argparse.RawTextHelpFormatter)
+
+    parser.add_argument("-m", "--model", help="Model", action="store_true")
+    parser.add_argument("model_name", type=str, help="Model ID")
+    parser.add_argument("-d", "--diagram", help="Diagram in the Trivium model", action="store_true")
+    parser.add_argument("diagram_name", type=str, help="Diagram ID")
+    parser.add_argument("-n", "--nessus", help="Performs a nessus scan with a list of files containing IP addresses", action="store_true")
+    parser.add_argument("nessus_list", nargs='?', type=str, help="List of Files containing IP addresses")
```
Comment (ethanizen) [inline distill/distill.py]:
```suggestion
    # initialize parser
    parser = argparse.ArgumentParser()

    # display landing screen for command-line tool
    parser.add_argument("-m", "--model", type=str, help="Model Name", required=True)
    parser.add_argument("-d", "--diagram", type=str, help="Diagram Name", required=True)
    parser.add_argument("-n", "--nessus", nargs='+', type=str, help="Nessus Files", required=True)

```

**Review by pwhph1 — APPROVED** [review]
Noice

**Pull request merged**